cime_config/acme/machines/template.case.run

```diff
@@ -226,16 +226,20 @@ sub doPreRunChecks()
 # -------------------------------------------------------------------------
 sub runModel()
 {
-    chdir $config{'RUNDIR'};
-
     system("sleep 10");
     my $localtime = localtime;
     # Set OMP_NUM_THREADS
     $ENV{'OMP_NUM_THREADS'} = {{ thread_count }};
 
     $logger->info("$localtime MODEL EXECUTION BEGINS HERE");
     # Run the model
-    {{ mpirun }}
```

Comment:
Some of the variables in the {{ mpirun }} statement cannot be resolved until you are on the compute node, this change delays getting the resolved values of variables to solve that problem.